app/src/main/java/com/del/delcontainer/ui/settings/SettingsViewModel.java

```diff
@@ -1,19 +1,61 @@
 package com.del.delcontainer.ui.settings;
 
+import android.util.Log;
+
 import androidx.lifecycle.LiveData;
 import androidx.lifecycle.MutableLiveData;
 import androidx.lifecycle.ViewModel;
+import com.del.delcontainer.database.entities.UserProfile;
+import com.del.delcontainer.repositories.UserProfileRepository;
+import com.del.delcontainer.ui.login.LoginStateRepo;
+import com.del.delcontainer.utils.apiUtils.APIUtils;
+import com.del.delcontainer.utils.apiUtils.interfaces.UserApi;
+import com.del.delcontainer.utils.apiUtils.pojo.UserDetails;
 
-public class SettingsViewModel extends ViewModel {
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+import retrofit2.Retrofit;
 
-    private MutableLiveData<String> mText;
+public class SettingsViewModel extends ViewModel {
 
-    public SettingsViewModel() {
-        mText = new MutableLiveData<>();
-        mText.setValue("User Name");
+    private static final String TAG = "SettingsViewModel";
+    private MutableLiveData<String> firstName = new MutableLiveData<>();
+    private MutableLiveData<LoginStateRepo> loginStateRepo = new MutableLiveData<>();
+    public MutableLiveData<LoginStateRepo> getLoginStateRepo() {
+        return loginStateRepo;
```

Comment:
What are you trying to do here?

Reply:
Sorry, I forgot to remove the 24 and 25. They’re not doing anything. I’ll delete them as well.